platform-configs/msi-pro-z690-a-ddr5.robot

```diff
@@ -192,6 +192,9 @@ ${NETBOOT_UTILITIES_SUPPORT}=                       ${FALSE}
 ${PETITBOOT_PAYLOAD_SUPPORT}=                       ${FALSE}
 ${HIBERNATION_AND_RESUME_SUPPORT}=                  ${FALSE}
 ${THUNDERBOLT_DOCKING_STATION_DETECT_SUPPORT}=      ${FALSE}
+${PLATFORM_CPU_SPEED}=                              3.50
```

Comment:
@filipleple We must add these to all configs, or add a flag preventing these tests from execution on unsupported platforms. You can probably simplify it by checking if we have non-zero value, that would be a good start. This way, no need to add these flags at all.

Reply:
Added an [exception](https://github.com/Dasharo/open-source-firmware-validation/pull/204/commits/6130e7558a7fa3957a97b77f48047f20354b73bc#diff-5d1c440a4db61d6667255f7b98667e28015156f4d50e3015b504ca408598bdf7R80) that skips expected value cases if the variables are not set

Reply:
@filipleple I have seen that. I am not fully convinced on this solution, though.

It seems we have to repeat these four lines multiple times whenever we want to skip some test:

```
    ${flags}=    Get Variables
    ${out}=    Run Keyword And Return Status
    ...    Should Be True    "\${PLATFORM_RAM_SPEED}" in $flags
    IF    not ${out}    Skip    Expected RAM speed undefined
```

This could be the keyword, at least. Any other ideas how to solve it?

Reducing configs duplication as in: https://github.com/Dasharo/open-source-firmware-validation/pull/210 helps here in any way?